store/policy.go

```diff
@@ -300,6 +405,59 @@ func (s *Store) upsertPolicyImpl(ctx context.Context, tx *sql.Tx, upsert *api.Po
 		&policyRaw.CreatedTs,
 		&policyRaw.UpdaterID,
 		&policyRaw.UpdatedTs,
+		&policyRaw.RowStatus,
+		&policyRaw.EnvironmentID,
+		&policyRaw.Type,
+		&policyRaw.Payload,
+	); err != nil {
+		return nil, FormatError(err)
+	}
+
+	return &policyRaw, nil
+}
+
+// patchPolicyImpl updates an existing policy by id and type.
+func (s *Store) patchPolicyImpl(ctx context.Context, tx *sql.Tx, patch *api.PolicyPatch) (*policyRaw, error) {
```

Comment:
looks like we don't need the s *Store, same for delete

Reply:
Just want to align with the existed `upsertPolicyImpl` and `findPolicyImpl` functions. Or shall we also remove the `(s *Store)` for them?

Reply:
yes, I checked the other places and looks like without s is the convention